core/drivers/clk/clk.c

```diff
@@ -107,6 +110,11 @@ TEE_Result clk_register(struct clk *clk)
 	return TEE_SUCCESS;
 }
 
+const char *_clk_elt_name(struct clk *clk)
```

Comment:
I don't see the point with the leading `_`.

Reply:
ok